lua/net.lua

```diff
@@ -1,10 +1,14 @@
 local gatekeeperc = require("gatekeeperc")
+local ffi = require("ffi")
 
 local M = {}
 
 -- Function that sets up the network.
 function M.setup_block()
 
+	local front_ports = {"0000:83:00.0"}
+	local back_ports = {"0000:85:00.0"}
```

Comment:
Isn't the ids that DPDK uses to interfaces the same that the kernel uses? If so, all you need to do is to get the id the kernel uses; that's exactly what `xip` does.